G4_DailyReport/src/main/java/com/example/G4_DailyReport/controller/admin/DepartmentController.java

```diff
@@ -84,8 +78,27 @@ public String edit(Model model, @PathVariable UUID id) {
     }
 
     @GetMapping("/{id}/show")
-    public String show(Model model, @PathVariable UUID id) {
-        return null;
+    public String show(
+            Model model,
+            @PathVariable UUID id,
+            @RequestParam("page") Optional<Integer> page,
+            @RequestParam("size") Optional<Integer> size
+    ) {
+        Pageable pageable = PageRequest.of(
+                page.orElse(1) - 1,
+                size.orElse(5),
+                Sort.by("id").ascending()
+        );
+        Department department = departmentService.findById(id);
+        Page<User> managers = departmentService.findManagers(id, pageable);
+
+        if(department == null) {
+            return "redirect:/admin/error";
+        }
```

Comment:
```suggestion
        if(department == null) {
            return "redirect:/admin/error";
        }
        Page<User> managers = departmentService.findManagers(id, pageable);

```
nếu đã trả về error thì ko cần load managers làm j bé nha

Reply:
Đã sửa ạ